react.js

```diff
@@ -5,4 +5,9 @@ module.exports = {
   extends: [
     'react-app',
   ],
+
+  rules: {
+    // Allow methods that do use `this`
```

Comment:
I think what the comment should be is ``"Allow class methods that do not use `this`"``.